proxy/http3/Http3Frame.cc

```diff
@@ -261,9 +284,11 @@ Http3SettingsFrame::Http3SettingsFrame(const uint8_t *buf, size_t buf_len) : Htt
   }
 }
 
-void
-Http3SettingsFrame::store(uint8_t *buf, size_t *len) const
+int64_t
+Http3SettingsFrame::store(QUICStreamIO *stream_io)
 {
+  // header + payload
+  uint8_t header[128]                                   = {0};
```

Comment:
`Http3Frame::MAX_FRAM_HEADER_OVERHEAD` ?

Reply:
get it